source/docs/deploying.md

```diff
@@ -1,13 +1,75 @@
 title: Deploying DApps to IPFS and Swarm
 ---
+To upload your Dapp to IPFS or Swarm, first ensure you have correctly set up your dApp's `storage.json` config to specify the preferred [storage configuration settings](./storage.html)</a>, namely `provider`, `host`, `port`, and `getUrl`.
+
+Once the config is set, run
+```
+embark upload
+```
+If you want to deploy to the livenet then after configuring you account on `config/blockchain.json` on the `livenet` environment then you can deploy to that chain by specifying the environment: 
```

Comment:
Not sure what that sentence means. Also, "you**r** account".

Reply:
Just saw you didn't write this. Was probably @iurimatias . What does that mean? ☝️

Reply:
I think he means configuring an ethereum account (address/seed phrase) in blockchain config?? @iurimatias ?  Probably would be be good to get this merged since we are not using `embark upload platform` any longer.

Reply:
the long version is:

After specifying the "account" field in `config/blockchain.json` for an environment like `livenet` https://github.com/embark-framework/embark/blob/develop/templates/demo/config/blockchain.json#L45 the user can then when they do embark upload it will build and deploy to this environment which is configured to be the livenet, (these days it's being called 'mainnet') and they can do this by doing ```embark upload livenet```

A lot of this might be outdated now that we support the accounts stuff in contracts.json

Reply:
@iurimatias, makes sense. Is there anything you'd like me to change in the documentation? Or can this be merged?

Reply:
Well, I guess you could at least correct "you account" for "your account". Then, something like this sounds better IMO:
"If you want to deploy yo the livenet, just configure your account in `config/blockchain.json` under the  `livenet` environment. Then, you can deploy to that chain with that command:"

Reply:
Fixed.